start.html

```diff
@@ -99,7 +99,49 @@
 how to find elements on the page. WebDriver offers a number of built-in selector
 types, amongst them finding an element by its ID attribute:
 
-<pre><code class=java>WebElement cheese = driver.findElement(By.id("cheese"));</code></pre>
+    <div class="tabs">
+        <ul class="tabs">
+            <li><a href="#java">java</a></li>
+            <li><a href="#ruby">ruby</a></li>
+            <li><a href="#python">python</a></li>
+            <li><a href="#c-sharp">c#</a></li>
+            <li><a href="#perl">perl</a></li>
+            <li><a href="#php">php</a></li>
+            <li><a href="#jscript">javascript</a></li>
```

Comment:
I wonder if we can use a customa ttribute `data-lang` to define the language. That seems more appropriate than hijacking anchor names.